lib/config/cmd-list.js

```diff
@@ -28,9 +28,8 @@ var affordances = {
   'verison': 'version',
   'ic': 'ci',
   'innit': 'init',
-  'isntall': 'install',
+  'install': 'install',
```

Comment:
this is very much not a typo; it's an attempt for npm to fix a *user's* typo.

Reply:
Ok. Thanks for the answer.